downstream/modules/eda/con-custom-credential-types.adoc

```diff
@@ -0,0 +1,58 @@
+[id="eda-custom-credential-types"]
+
+= Custom credential types
+
+As a system administrator, you can define a custom credential type works in ways similar to existing credential types in a standard format using a YAML or JSON-like definition. 
+
+Each credential type displays its own unique configurations in the Input Configuration field and the Injector Configuration field, if applicable. Custom credentials support Ansible extra variables as a means of injecting their authentication information. 
+
+//[J.Self] Reviewers: please confirm if this is true. I borrowed it from automation controller content, per suggestions from several SMEs and one of our principal tech writers. I just want to confirm technical accuracy here, especially the last line as we might not have "machine credentials".
+You can attach one or more cloud, vault, and {PlatformName} credentials to a job template. 
```

Comment:
@jself-sudoku 
to a job template should be to Activation. We don't have Job Templates in EDA, they are from AAP

Reply:
Thanks! Confirming the update.